src/Sarif/VersionConstants.cs

```diff
@@ -5,8 +5,8 @@ namespace Microsoft.CodeAnalysis.Sarif
     public static class VersionConstants                                       
     {                                                                          
         public const string Prerelease = "-beta";                       
-        public const string AssemblyVersion = "1.5.21";       
-        public const string FileVersion = "1.5.21" + ".0";    
+        public const string AssemblyVersion = "1.5.22";       
+        public const string FileVersion = "1.5.22" + ".0";    
```

Comment:
How'd this get updated? Why don't I see a chance somewhere to SetCurrentVersion.cmd?